client/app/components/queries/SchemaBrowser.jsx

```diff
@@ -70,18 +71,25 @@ function SchemaItem({ item, expanded, onToggle, onSelect, ...props }) {
           {item.loading ? (
             <div className="table-open">Loading...</div>
           ) : (
-            map(item.columns, column => {
+            map(item.columns, (column) => {
               const columnName = get(column, "name");
               const columnType = get(column, "type");
+              const columnDescription = get(column, "description");
               return (
                 <Tooltip
-                  title="Insert column name into query text"
+                  title={"Insert column name into query text" + (columnDescription ? "\n" + columnDescription : "")}
                   mouseEnterDelay={0}
                   mouseLeaveDelay={0}
-                  placement="rightTop">
-                  <PlainButton key={columnName} className="table-open-item" onClick={e => handleSelect(e, columnName)}>
+                  placement="rightTop"
+                >
+                  <PlainButton
+                    key={columnName}
+                    className="table-open-item"
+                    onClick={(e) => handleSelect(e, columnName)}
+                  >
                     <div>
                       {columnName} {columnType && <span className="column-type">{columnType}</span>}
+                      {columnDescription && <span className="column-description"> - {columnDescription}</span>}
```

Comment:
Potentially very useful. I feel that there often is not enough room to fit a column comment. Example:

> atime  TIMESTAMP WITH TIME ZONE - Last acc...

Adding the comment to  the Tooltip may be enough

_Maybe_ add a toggle to show datatype or column description